app/Http/Controllers/UsersController.php

```diff
@@ -45,7 +70,22 @@ public function store(Request $request)
     public function update(Request $request, $id)
     {
         $user = User::find($id);
-        $user->update($request->all());
+        $user->name = $request->input('name');
+        $user->address = $request->input('address');
+        $user->age = $request->input('age');
+        // $user->photo = $request->input('photo');
+        if($request->file('photo')!==null) :
+            $image = $request->file('photo');
+            $imageName = time().'.'.$image->getClientOriginalExtension();
+            $destinationPath = public_path('/img');
+            $img = Image::make($image->getRealPath())
```

Comment:
You're mixing image and img. You'd better unify which. Also I don't recommend short word for variables because it just decreases readability.